tests/zfs-tests/tests/functional/cli_root/zpool_initialize/cleanup.ksh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/ksh -p
```

Comment:
This needs to be `/bin/ksh -p` and it's what caused the CentOS 6 failure.

Reply:
Fixed this in https://github.com/zfsonlinux/zfs/pull/7954